install/wp-site

```diff
@@ -36,20 +46,27 @@ chown -R $userName:www-data $baseDir
 chmod -R 750 $baseDir
 chmod -R g+s $baseDir
 
-echo "Installing Wordpress + AD-plugin"
-# Install Wordpress + AD plugin using wp-cli
+echo "Installing Wordpress"
+# Install Wordpress 
 su - $userName -c "
 cd $installDir
 wp core download --locale=sv_SE
 wp core config --dbname=$userName --dbuser=$userName --dbpass=$userPassword --locale=sv_SE
-wp core install --url="https://$FQDN" --title='$userName website' --admin_user=$adminUser --admin_password=$wpAdminPassword --admin_email=$adminMail
+wp core install --url="https://$FQDN" --title='$userName website' --admin_user=$adminUser --admin_password=$wpAdminPassword --admin_email=$adminMail"
+
+# Installing AD plugin for Wordpress
+echo "Installing AD-plugin"
+if [[ $active_directory == "yes" ]]
+then
+    su $username -c "cd $installDir
 wp plugin install active-directory-integration
 wp plugin activate active-directory-integration"
+fi
```

Comment:
If this is all we do, why don't we let the user provide an array of plugins to be installed instead of hard-coding it? We could have active-directory-integration as an example in the manual.